src/main/java/subsystems/wrist/Wrist.java

```diff
@@ -0,0 +1,27 @@
+package subsystems.wrist;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Wrist extends SubsystemBase {
+
+    private static Wrist instance;
+    private static TalonSRX motor;
+
+    private Wrist() {
+        motor = new TalonSRX(Constants.WRIST_MOTOR_ID);
+    }
+
+    public static Wrist getInstance() {
+        if (instance == null) {
+            instance = new Wrist();
+        }
+        return instance;
+    }
+
+    public void getToPosition(Rotation2d position) {
```

Comment:
consider changing from "get" to "go". get is associated with a return type, while this function is void.
(same in elbow)